.github/workflows/test.yml

```
@@ -92,7 +92,8 @@ jobs:
       - test
     runs-on: ubuntu-latest
```

I think it's probably easier to just disable this on forks with something like (might need to move it into the steps)

```suggestion
    runs-on: ubuntu-latest
    if: github.repository == 'hackingmaterials/matminer'
```

This part of the release doesn't seem to be used much; in the past I've just manually made the release and its all seemed to work.